src/spell_check.c

```diff
@@ -139,16 +139,15 @@ static void SpellCheck_FindSuggestions(SpellCheckCtx *scCtx, Trie *t, const char
     double score;
     if ((score = SpellCheck_GetScore(scCtx, res, suggestionLen, fieldMask)) != -1) {
       RS_SuggestionsAdd(s, res, suggestionLen, score, incr);
-    } else {
-      free(res);
     }
+    free(res);
   }
   DFAFilter_Free(it->ctx);
   free(it->ctx);
   TrieIterator_Free(it);
 }
 
-RS_Suggestion** spellCheckReducerTerm_GetSuggestions(RS_Suggestions *s){
+RS_Suggestion** spellCheck_GetSuggestions(RS_Suggestions *s){
```

Comment:
seems like this should be static?

Reply:
No, I am using it from on the external module (coordinator)